modules/gateways/blockonomics/assets/templates/checkout.tpl

```diff
@@ -21,25 +18,25 @@
             <div class="bnomics-spinner-wrapper" ng-show="spinner" ng-cloak>
                 <div class="bnomics-spinner"></div>
             </div>
-            <!-- Address Error -->
-            <div id="address-error-btc" ng-show="address_error_btc" ng-cloak>
+            <!-- BTC Address Error -->
+            <div id="address-error-btc" ng-show="address_error['btc']" ng-cloak>
                 <h2>{$_BLOCKLANG.error.btc.title}</h2>
                 <p>{$_BLOCKLANG.error.btc.message}</p>
             </div>
             <!-- BCH Address Error -->
-            <div id="address-error-bch" ng-show="address_error_bch" ng-cloak>
+            <div id="address-error-bch" ng-show="address_error['bch']" ng-cloak>
                 <h2>{$_BLOCKLANG.error.bch.title}</h2>
                 <p>{$_BLOCKLANG.error.bch.message}</p>
             </div>
-            <!-- Pending payment -->
-            <div id="pending-error" ng-show="pending_error" ng-cloak>
-                <h2>{$_BLOCKLANG.error.pending.title}</h2>
-                <i>{$_BLOCKLANG.error.pending.message}</i>
+            <!-- Gap limit + Duplicate Address Error -->
+            <div id="address-error-message" ng-show="customer_facing_error" ng-cloak>
+                <h2>{$_BLOCKLANG.error.addressGeneration.title}</h2>
+                <p>[[customer_facing_error]]</p>
             </div>
             <!-- Payment Expired -->
             <div class="bnomics-order-expired-wrapper" ng-show="order.status == -3" ng-cloak>
                 <h3 class="warning bnomics-status-warning">{$_BLOCKLANG.paymentExpired}</h3><br>
-                <p><a href="#" ng-click="try_again_click()">{$_BLOCKLANG.tryAgain}</a></p>
+                <p><a ng-click="try_again_click()">{$_BLOCKLANG.tryAgain}</a></p>
```

Comment:
The try again link does not show cursor pointer on hover. Let's fix this behavior.